### PR TITLE
New Cal csvs for Irminger 6 and Pioneer 13 cruises

### PR DESCRIPTION
**Pull request by rtravis-whoi**
This PR includes 6 new METBKA cal csvs for the surface moorings on both the Irminger 6 and Pioneer 13 cruises.

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
File for reference (new csvs highlighted at bottom of file)
[METBK_Cal_Updates_2019-11-07.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/3821762/METBK_Cal_Updates_2019-11-07.xlsx)

File: calibration/METBKA/CGINS-METLGR-00010__20190927.csv [hunk 1]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR010,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP01CNSM-00012
+LGR010,CC_height_of_air_humidity_measurement_m,4.255,
+LGR010,CC_height_of_air_temperature_measurement_m,4.255,
+LGR010,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR010,CC_jcool,1,Constant
+LGR010,CC_jwarm,1,Constant
+LGR010,CC_zinvpbl,600,Constant
+LGR010,CC_use_velpt,1,
```
Comment (cdobs) [inline calibration/METBKA/CGINS-METLGR-00010__20190927.csv]: Looks good

File: calibration/METBKA/CGINS-METLGR-00012__20190926.csv [hunk 2]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR012,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP03ISSM-00011
+LGR012,CC_height_of_air_humidity_measurement_m,4.255,
+LGR012,CC_height_of_air_temperature_measurement_m,4.255,
+LGR012,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR012,CC_jcool,1,Constant
+LGR012,CC_jwarm,1,Constant
+LGR012,CC_zinvpbl,600,Constant
+LGR012,CC_use_velpt,1,
```
Comment (cdobs) [inline calibration/METBKA/CGINS-METLGR-00012__20190926.csv]: Looks good

File: calibration/METBKA/CGINS-METLGR-00017__20190927.csv [hunk 3]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR017,CC_depth_of_conductivity_and_temperature_measurements_m,1.11,CP01CNSM-00012
+LGR017,CC_height_of_air_humidity_measurement_m,4.255,
+LGR017,CC_height_of_air_temperature_measurement_m,4.255,
+LGR017,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR017,CC_jcool,1,Constant
+LGR017,CC_jwarm,1,Constant
+LGR017,CC_zinvpbl,600,Constant
+LGR017,CC_use_velpt,1,
```
Comment (cdobs) [inline calibration/METBKA/CGINS-METLGR-00017__20190927.csv]: Looks good

File: calibration/METBKA/CGINS-METLGR-00037__20190929.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR037,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP04OSSM-00011
+LGR037,CC_height_of_air_humidity_measurement_m,4.255,
+LGR037,CC_height_of_air_temperature_measurement_m,4.255,
+LGR037,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR037,CC_jcool,1,Constant
+LGR037,CC_jwarm,1,Constant
+LGR037,CC_zinvpbl,600,Constant
+LGR037,CC_use_velpt,1,
```
Comment (cdobs) [inline calibration/METBKA/CGINS-METLGR-00037__20190929.csv]: Looks good

File: calibration/METBKA/CGINS-METLGR-00027__20190805.csv [hunk 5]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR027,CC_depth_of_conductivity_and_temperature_measurements_m,1.23,GI01SUMO-00006
+LGR027,CC_height_of_air_humidity_measurement_m,5.255,
+LGR027,CC_height_of_air_temperature_measurement_m,5.255,
+LGR027,CC_height_of_windspeed_sensor_above_sealevel_m,5.74,
+LGR027,CC_jcool,1,Constant
+LGR027,CC_jwarm,1,Constant
+LGR027,CC_zinvpbl,600,Constant
+LGR027,CC_use_velpt,1,
```
Comment (cdobs) [inline calibration/METBKA/CGINS-METLGR-00027__20190805.csv]: Looks good

File: calibration/METBKA/CGINS-METLGR-00036__20190805.csv [hunk 6]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR036,CC_depth_of_conductivity_and_temperature_measurements_m,1.37,GI01SUMO-00006
+LGR036,CC_height_of_air_humidity_measurement_m,5.255,
+LGR036,CC_height_of_air_temperature_measurement_m,5.255,
+LGR036,CC_height_of_windspeed_sensor_above_sealevel_m,5.74,
+LGR036,CC_jcool,1,Constant
+LGR036,CC_jwarm,1,Constant
+LGR036,CC_zinvpbl,600,Constant
+LGR036,CC_use_velpt,1,
```
Comment (cdobs) [inline calibration/METBKA/CGINS-METLGR-00036__20190805.csv]: Looks good

**Pull request merged**